src/patternfly/components/Nav/nav.scss

```diff
@@ -489,6 +491,18 @@
     }
   }
 
+  .pf-c-menu__list-item:where(.pf-m-drill-up) {
```

Comment:
Curious what `:where()` is used for here - is it to maintain specificity or something else?